styles.css/main.css

```diff
@@ -144,39 +151,45 @@ ol {
 table {
   margin-left: auto;
   margin-right: auto;
-  padding-top: 3em;
-  padding-bottom: 3em;
-  padding-left: 1em;
-  padding-right: 1em;
   color: #fff;
-  background-color:rgb(0, 0, 0);opacity:0.5;
   border-collapse: collapse;
   text-align: left;
   
   
 }
 
-tr
+td,
+th {
+  padding-top: 0.3em;
+  padding-bottom: 0.3em;
+  padding-left: 1em;
+  padding-right: 1em;
+}
+
+tbody tr td
 {
-  background-color: rgba(0, 0, 0, 0.5);
+  background-color: rgba(0, 0, 0, 0.05);
   
 }
 
-tr:nth-child(even) {background-color: #0d3e77;}
+tbody tr:nth-child(even) td {background-color: rgba(0,0,0,0.15); }
 
 
 
 
-thead{
+thead th,
+thead td {
  
   color: #fff;
-  background-color:rgb(0, 0, 0);opacity:0.2;
+  background-color:rgb(0, 0, 0, 0.2);
```

Comment:
To make colors transparent, use the `rgba` function and set the alpha (a) value to a number between 0 and 1